common-requirements.md

```diff
@@ -0,0 +1,29 @@
+## Title
+Common Requirements
+
+## Context
+Many projects are trying to use common code.  There is a shared repository that all the projects access.  This pattern applies if there is a Strong Code Owner [pattern to be written] or if there is weak code ownership, or no Benevolent Sponsor [pattern to be written]. Someone (or some project) wrote the code in the first place and contributed it to the repository. The common code is a small percentage of the overall deliverable from any of the projects.  Each project has its own delivery schedule, set of deliverables and customers.  
+Statement of Problem:  The common code in the shared repository isn’t meeting the needs of all the projects that want to use it.  
+
+## Problem Statement
+The common code in the shared repository isn’t meeting the needs of all the projects that want to use it.  
+
+## Forces
```

Comment:
Are there any relevant forces related to the human factor in all this we should consider? For me personally, InnerSource is much more about culture and the human factor than it is about code and processes.